src/cmake/externalpackages.cmake

```diff
@@ -163,7 +163,7 @@ checked_find_package (GIF
                       VERSION_MIN 4
                       RECOMMEND_MIN 5.0
                       RECOMMEND_MIN_REASON "for stability and thread safety")
-checked_find_package (Libheif VERSION_MIN 1.3)  # For HEIF/HEIC format
+checked_find_package (Libheif VERSION_MIN 1.3)  # For HEIF/HEIC/AVIF formats
```

Comment:
# For HEIF/HEIC/AVIF formats
    checked_find_package (Libheif VERSION_MIN 1.3
                          RECOMMENDED_MIN 1.7
                          RECOMMENDED_MIN_REASON "for AVIF support")